particle.js

```diff
@@ -66,19 +66,6 @@ const cssModes = {
   },
 };
 
-/**
- * Polyfill ALL modern JS to support for browsers within .browserslistrc
- *
- * @param {string} entry
- * @returns {{entry: {}}}
- */
-const entryPrepend = entry => ({
-  // See webpack.[app].js for more entry points
-  entry: {
-    [entry]: ['@babel/polyfill'],
```

Comment:
Trying to follow along...

This is due to https://babeljs.io/docs/en/babel-preset-env#usebuiltins

Reply:
Exactly! We no longer need to sneak in this entry point. Used to be super hacky, no longer hacky.